src/GovUk.Education.ExploreEducationStatistics.Data.Api.Tests/TestStartup.cs

```diff
@@ -0,0 +1,72 @@
+#nullable enable
+using GovUk.Education.ExploreEducationStatistics.Common.ModelBinding;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
```

Comment:
Although technically unused, it's actually being referenced in the docblock. Otherwise, we'd have to use a really ugly reference path of `GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures.TestApplicationFactory{TStartup}`.

It's been discussed in retros before, but it really would be nice if we had a much shorter namespace like `GovUk.Education`.

Reply:
Ah, yeah, I didn't see that, thanks, and good point about the namespace.